src/rp2_common/hardware_watchdog/include/hardware/watchdog.h

```diff
@@ -80,6 +86,23 @@ void watchdog_enable(uint32_t delay_ms, bool pause_on_debug);
  */
 bool watchdog_caused_reboot(void);
 
+/**
+ * \brief Did watchdog_enable cause the last reboot?
+ * \ingroup hardware_watchdog
+ *
+ * Perform additional checking along with \ref watchdog_caused_reboot to determine if a watchdog timeout initiated by
+ * \ref watchdog_enable caused the last reboot.
+ *
+ * This method checks for a special value in watchdog scratch register 4 placed there by \ref watchdog_enable.
+ * This would not be present if a watchdog reset is initiated by \ref watchdog_reboot or by the RP2040 bootrom
+ * (e.g. dragging a UF2 onto the RPI-RP2 drive).
+ *
+ * @return true if the watchdog timer or a watchdog force caused (see \reg watchdog_caused_reboot) the last reboot
```

Comment:
Does #561 mean that this "watchdog force" part is no longer true? Which I guess would mean this could be shortened to
`@return true if the watchdog timer caused a reboot after \ref watchdog_enable was called` which IMHO seems much more readable? :shrug: